src/frontend-admin/src/components/reusables/Card/Card.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import styles from './Card.module.css'
+import {Title} from '../Title/Title' 
+
+interface ICardProps {
+    title: any
```

Comment:
Nice, I agree this should be an `any`. That way we can pass in strings, components, whatever

Reply:
That said, this likely should be `title?: any`. That'll give us flexibility to have cards without titles if we ever need them.